KeywordSearch/src/org/sleuthkit/autopsy/keywordsearch/RawText.java

```diff
@@ -262,33 +259,39 @@ private String getContentText(int currentPage, boolean hasChunks) throws SolrSer
         }
 
         //not cached
-        try {
-            String indexedText = solrServer.getSolrContent(this.objectId, chunkId);
-            if (indexedText == null) indexedText = "";
-            cachedString = EscapeUtil.escapeHtml(indexedText).trim();
-            StringBuilder sb = new StringBuilder(cachedString.length() + 20);
-            sb.append("<pre>").append(cachedString).append("</pre>"); //NON-NLS
-            cachedString = sb.toString();
-            cachedChunk = chunkId;
-        } catch (NoOpenCoreException ex) {
-            logger.log(Level.SEVERE, "No open core", ex); //NON-NLS
-            return "";
+        String indexedText = solrServer.getSolrContent(this.objectId, chunkId);
+        if (indexedText == null || indexedText.isEmpty()) {
+            return NbBundle.getMessage(ExtractedContentViewer.class, "ExtractedContentViewer.getSolrContent.noTxtYetMsg", "file");
```

Comment:
For a file, null should lead to an error message and empty text to the no text message (see above).